File: REST_DotNET_Coffee_Android/DTO/CartDTO/CartItemResponseDTO.cs
#nullable disable
public class CartItemResponseDTO
{
    public int ItemId { get; set; }
    public int ProductId { get; set; }

    public int Quantity { get; set; }

    public List<String> IngredientList { get; set; }

    public double PreTotal { get; set; }
}

File: Mobile_Coffee/Project-Mobile__Coffe-Store/app/src/main/java/com/nlu/packages/dotnet_callapi/responsedto/CartItemResponseDTO.java
package com.nlu.packages.dotnet_callapi.responsedto;

import java.io.Serializable;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CartItemResponseDTO implements Serializable {
    private int itemId;
    private int productId;
    private int quantity;
    private List<String> ingredientList;
    private double preTotal;
}
